README.md

```diff
@@ -60,7 +60,46 @@ Simply `npm install vue-fineuploader@next` and see the documentation for your sp
 
 #### `<gallery />`
 
-Not implemented yet.
+```javascript
+<template>
+  <Gallery :uploader="uploader" />
+</template>
+
+<script>
+  import FineUploaderTraditional from 'fine-uploader-wrappers';
+  import Gallery from '../modules/vue-fineuploader/src/gallery';
```

Comment:
Could you remove the `;` here ? For consistency.

Also, `Gallery` should be loaded from 'vue-fineuploader/gallery'. (I have to update the README.md for all the other components)